docs/working-with-cdk-for-terraform/using-modules.md

```diff
@@ -0,0 +1,102 @@
+# Using Modules
```

Comment:
Maybe name it `Terraform Modules` in the heading to make clear that this is not about the "modules" one would use for organizing a CDKTF project (which would rather make use of custom constructs) instead.